### PR TITLE
Remove CXF client and update vulnerability suppressions

### LINKED ISSUE #18: Remove CXF Client
**Describe the task**
Remove the CXF client from the repository until we have standard way to manage API clients and SDKs.

**Additional context**
Generating clients at this point is counter productive as they are not being used, the code generated is not of the highest quality, and may not even be the type of clients we want to expose and support moving forward.

### PR DESCRIPTION
**Pull request by lessarderic**
#### What does this PR do?
Removes the CXF client module so it doesn't get generated anymore.
Also, moves the OWASP suppression files under the `owasp` directory and updates the suppressions.

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@emmberk 
@paouelle 
@jhunzik 
@stustison 
@ahoffer 

#### How should this be tested? (List steps with links to updated documentation)
Successful build.

#### What are the relevant tickets?
Fixes #18 

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by lessarderic** [comment]
build now

File: rest/models/owasp/suppressions.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+-->
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.2.xsd">
+    <suppress>
+        <!--
+            OWASP infers that jackson-databind-nullable 0.1.0 is related to Jackson 0.1.0
+            dependencies and flags vulnerabilities on cpe:2.3:a:fasterxml:jackson:0.1.0 and
+            cpe:2.3:a:fasterxml:jackson-databind:0.1.0, which do not exist.
+            GitHub issue https://github.com/jeremylong/DependencyCheck/issues/2047 was opened to
+            get this vulnerability resolved in OWASP/DependencyCheck itself. This suppression should
+            be removed when that issue has been resolved.
```
Comment (paouelle) [inline rest/models/owasp/suppressions.xml]: 😆 You just made my day with this great comment. Thanks

Comment (paouelle) [reply rest/models/owasp/suppressions.xml]: ❓ Should we start adding a history with each suppression? Like the date it was added and the above comment. That way, when it needs to be revisited later by a developer, they can add to it indicating it is still ok and we will be able to track down how often a given suppression has been re-analyzed and by whom using git blame (assuming nobody else touches that particular line)

File: rest/servers/spring-boot/owasp/suppressions.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  ~ See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+  ~ <http://www.gnu.org/licenses/lgpl.html>.
+  -->
+<!--
+    Contains the suppressions that should not be inherited by dependent projects.
+-->
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.2.xsd">
+    <suppress>
+        <!--
+            OWASP infers that jackson-databind-nullable 0.1.0 is related to Jackson 0.1.0
```
Comment (paouelle) [inline rest/servers/spring-boot/owasp/suppressions.xml]: Same comments as before.

File: rest/servers/spring/owasp/suppressions.xml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  ~ See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+  ~ <http://www.gnu.org/licenses/lgpl.html>.
+  -->
+<!--
+    Contains the suppressions that should not be inherited by dependent projects.
+-->
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.2.xsd">
+    <suppress>
+        <!--
+            OWASP infers that jackson-databind-nullable 0.1.0 is related to Jackson 0.1.0
```
Comment (paouelle) [inline rest/servers/spring/owasp/suppressions.xml]: Same comments as before.

**Comment by lessarderic** [comment]
build now

**Pull request merged**